apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs

```diff
@@ -5,34 +5,45 @@ defmodule Pipeline.Writer.TableWriterTest do
 
   alias Pipeline.Writer.TableWriter
   alias Pipeline.Writer.TableWriter.Compaction
+  alias Pipeline.Application
   alias SmartCity.TestDataGenerator, as: TDG
   import SmartCity.TestHelper, only: [eventually: 1]
 
+  setup do
+    session =
+      Application.prestige_opts()
+      |> Prestige.new_session()
+
+    [session: session]
+  end
+
   describe "init/1" do
-    test "creates table with correct name and schema" do
+    test "creates table with correct name and schema", %{session: session} do
       expected = [
         %{"Column" => "one", "Comment" => "", "Extra" => "", "Type" => "array(varchar)"},
         %{"Column" => "two", "Comment" => "", "Extra" => "", "Type" => "row(three decimal(18,3))"},
-        %{"Column" => "four", "Comment" => "", "Extra" => "", "Type" => "array(row(five integer))"}
+        %{"Column" => "four", "Comment" => "", "Extra" => "", "Type" => "array(row(five decimal(18,3)))"}
       ]
 
       schema = [
         %{name: "one", type: "list", itemType: "string"},
         %{name: "two", type: "map", subSchema: [%{name: "three", type: "decimal(18,3)"}]},
-        %{name: "four", type: "list", itemType: "map", subSchema: [%{name: "five", type: "integer"}]}
+        %{name: "four", type: "list", itemType: "map", subSchema: [%{name: "five", type: "decimal(18,3)"}]}
       ]
 
-      dataset = TDG.create_dataset(%{technical: %{systemName: "org_name__dataset_name", schema: schema}})
+      dataset = TDG.create_dataset(%{technical: %{systemName: "org_name_dataset_name", schema: schema}})
```

Comment:
Was removing this underscore intentional?

Reply:
i believe this is the naming convention so i'd be surprised if this isn't causing a test to fail ~Jeff